keyboards/mechlovin/infinity875/rev2/rules.mk

```diff
@@ -0,0 +1,4 @@
+# Build Options
+#   change yes to no to disable
+#
+DEFAULT_FOLDER = mechlovin/infinity875/rev1
```

Comment:
Im no expert but i think this should be part of  `infinity875/rules.mk` as in "when someone tries compiling without selecting a rev, default to 1", this would redirect rev2 compilations to rev1 files afaik

At that point this file should contain something like `# This file is intentionally left blank`, ie: the QMK compiling process needs to find a `rules.mk` to detect the folder as a keyboard, but you dont want any configuration on it

Reply:
Got it, my bad when I was put "DEFAULT_FOLDER" on rev2/rules.mk.
I also move it into infinity875/rules.mk.
Thank you.